src/vaadin-grid-data-provider-mixin.html

```diff
@@ -295,10 +299,12 @@
           }
 
           // Populate the cache with new items
+          const currentItems = Array.from(this.$.items.children).map(row => row._item);
+
```

Comment:
The placement of the above comment is now wrong. The comment should still be right above the forEach loop

Reply:
Done